content/blog/posts/2022-08-04-add-desktop-notifications.md

```diff
@@ -0,0 +1,241 @@
+---
+authorId: chuckcarpenter
+categories:
+  - notifications
+  - javascript
+date: ''
+slug: add-desktop-notifications
+title: Adding desktop notifications to your app/site
+---
+
+There are times when your application needs to inform users to act on something more
+timely than a normal notification (often known as toast notifications) may allow.
+If you need a user to do something and can't wait until they return to the application for them to notice, or just want to engage them when something new is available, desktop notifications might be a good fit for your app.
+
+Desktop notifications are supported in all modern browsers (sorry Internet Explorer, you're not invited). However, the user will have to opt in and they will also have to allow notifications in their OS settings. One they have opted in, you can notify them on the system level rather than just within your web browser page/tab.
```

Comment:
```suggestion
Desktop notifications are supported in all modern browsers (sorry Internet Explorer, you're not invited). However, the user will have to opt-in and they will also have to allow notifications in their OS settings. Once they have opted in, you can notify them on the system level rather than just within your web browser page/tab.
```